website/pages/docs/glossary/toc.md

```diff
@@ -0,0 +1,11 @@
+# Overview
+
+This is a collection of terms and use-cases that are used across CloudQuery documentation and blogs.
+
+- [What is The Modern Data Stack](./what-is-the-modern-data-stack)
+- [What is Infrastructure Data Lake](./what-is-infrastructure-data-lake)
+- [What is a CSPM](./what-is-cspm)
+- [What is a CIEM](./what-is-ciem)
+- [What is ASM (Attack Surface Management)](./what-is-asm)
+- [What is SOC2 Compliance](./soc2)
```

Comment:
```suggestion
- [What is The Modern Data Stack?](./what-is-the-modern-data-stack)
- [What is Infrastructure Data Lake?](./what-is-infrastructure-data-lake)
- [What is a CSPM?](./what-is-cspm)
- [What is a CIEM?](./what-is-ciem)
- [What is ASM?](./what-is-asm)
- [What is SOC2 Compliance?](./soc2)
```

We should be consistent with `_meta.json`. Also maybe we can drop the TOC page? The links are already available in the sidebar